src/screens/admin-home/components/header/styles.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { StyleSheet } from 'react-native';
+
+const useStyles = () => {
+  return useMemo(
```

Comment:
Why do we need this wrapper?

Reply:
It's new style rules that @R-Mielamud done 
![image](https://user-images.githubusercontent.com/70844218/160548452-b08e79d7-3a95-4cc8-a0ce-aff92d555486.png)

Reply:
Because StyleSheet is an object. It will cause many re-renders. Also, colors are hooks now. If they will appear here, useMemo will depend on them.